Tabs herwerken en functional header tabs

ksvlaanderen opened the pull request:
feat: UIG-2135 - tabs disable links
- VlTabs disable tab links
- VlTabs stories en story args verbeterd

feat: UIG-2343 - functional header tabs
- Functional header: storybook en cypress uitgebreid voor tabs
- Tabs: styling en gedrag voor gebruik in functional header

ksvlaanderen committed:
feat: UIG-2135 - tabs disable links

VlTabs disable tab links
VlTabs stories en story args verbeterd

Goldflow commented on libs/components/src/lib/functional-header/stories/vl-functional-header.stories.ts:
```diff
@@ -4,12 +4,16 @@ import { Meta, StoryFn } from '@storybook/web-components';
-        docs: { page: functionalHeaderDoc },
+        docs: {
+            page: functionalHeaderDoc,
+            transformSource: filterOutClasses,
```
had deze (`filterOutClasses`) nog niet gezien (maar herinner me dat je er van verteld had) - goede toevoeging!

zouden we deze mss in `apps/storybook/.storybook/preview.js` zetten? of beter voorzichtig enkel per story toepassen?

ksvlaanderen replied:
Ik heb hier ook over getwijfeld maar omdat ik niet weet wat er zich allemaal afspeelt in de codebase heb ik besloten om het voorzichtig te spelen en dit niet te doen.

Goldflow replied:
ok klinkt logisch en het veiligst, we laten dit zo doen dan

Goldflow reviewed (commented):
1 opmerking gemaakt, ik volg jou mening hierover

heb enkel diagonaal over code gelezen, zal later vandaag nog eens code lokaal uitchecken

ksvlaanderen committed:
feat: UIG-2343 - functional header tabs

Functional header: storybook en cypress uitgebreid voor tabs
Tabs: styling en gedrag voor gebruik in functional header

Goldflow commented on libs/components/src/lib/functional-header/stories/vl-functional-header.stories.ts:
```diff
@@ -72,6 +76,17 @@ FunctionalHeaderActions.args = {
+FunctionalHeaderTabs.storyName = 'vl-functional-header - tabs';
+FunctionalHeaderTabs.args = {
+    title: functionalHeaderArgs.title,
+    subHeaderSlot: `<vl-tabs slot="sub-header" data-vl-disable-links data-vl-within-functional-header data-vl-active-tab="trein">
```
vraag me af of het beter is `data-vl-active-tab="trein"` weg te laten; omdat de andere links ook niet actief worden en afnemers dat sowiesio zelf moeten implementeren (vermoed dat dit ergens vermeld staat)

Goldflow replied:
zoals besproken, laat gerust staan, gezien dit ook is hoe afnemer voor de eerste pagina

eventueel toe te lichten in storybook bij de aankomende task dat ze manueel de active tab moeten aanpassen afhankelijk van de pagina dat ze op zijn

Goldflow commented on libs/components/src/lib/functional-header/stories/vl-functional-header.stories-doc.mdx:
```diff
@@ -26,6 +26,12 @@ import { VlFunctionalHeaderComponent } from '@domg-lib/components';
 
+### Met tabs
+
+Voor meer informatie zie het [tabs](/story/components-tabs--tabs-default) component.
```
mss linken naar `/docs/` ipv `/story/`
